### PR TITLE
Soarian r4 procedure Resource Documentation

### PR DESCRIPTION
**Pull request by VamsiMure**
Description
Soarian FHIR R4 Procedure resource

PR Checklist
----
- [ ] Screenshot(s) of changes attached before changes merged.
procedure-1
procedure-2
procedure-3
procedure-4
procedure-5
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/soarian/r4/clinical/summary/procedure.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+---
+
+# Procedure
+
+* TOC
+{:toc}
+
+## Overview
+
+The Procedure resource implementation is limited to a patient’s known procedure history that occured at this site and is recorded in _Soarian Clinicals_<sup>®</sup>. As such, this resource may not be as complete as a procedure list provided as part of the patient’s billing record.
```
Comment (HRNWriter) [inline content/soarian/r4/clinical/summary/procedure.md:12]: Remove italic font from Soarian Clinicals.

PRODOC-160857.

File: content/soarian/r4/clinical/summary/procedure.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+_Soarian Clinicals_<sup>®</sup> supports a read-only Application Programming Interface (API). This API accepts `GET` and `POST` based [search] and `GET` based [read] interactions. The response represents the most current information about the patient that is charted in _Soarian Clinicals_<sup>®</sup> at the time of the query.
```
Comment (HRNWriter) [inline content/soarian/r4/clinical/summary/procedure.md:14]: Remove italic font from "Soarian Clinicals".

PRODOC-160857.

**Review by HRNWriter — COMMENTED** [review]
Please see my comments.
PRODOC-160857.

File: content/soarian/r4/clinical/summary/procedure.md (hunk 2 above)
Comment (VamsiMure) [reply content/soarian/r4/clinical/summary/procedure.md]: Done

File: content/soarian/r4/clinical/summary/procedure.md (hunk 1 above)
Comment (VamsiMure) [reply content/soarian/r4/clinical/summary/procedure.md]: Done

File: content/soarian/r4/clinical/summary/procedure.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+## Search
+
+Search for Procedure resources that meet specified query parameters:
+
+    GET /Procedure?:parameters
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true)%>
```
Comment (ShrutiNHegde) [inline content/soarian/r4/clinical/summary/procedure.md:41]: System token is missing

File: content/soarian/r4/clinical/summary/procedure.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+## Retrieve by ID
+
+List an individual Procedure resource by its ID:
+
+    GET /Procedure/:id
+
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true)%>
```
Comment (ShrutiNHegde) [inline content/soarian/r4/clinical/summary/procedure.md:96]: System token is missing

File: lib/resources/example_json/soarian_r4_examples_procedure.rb [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+      ],
+      "text": {
+        "status": 'generated',
+        # rubocop:disable Layout/LineLength
+        "div": '<div xmlns="http://www.w3.org/1999/xhtml"><b>Procedure</b><br /><b>Patient</b>:Lansford,Mukesh-Procedure Dianne;<br /><b>Encounter</b>:City General,Initial;<br /><b>Status</b>:entered-in-error;<br /><b>Code</b>:<br /><b>system</b>:http://snomed.info/sct <b>code</b>:429211003 <b>display</b>:Procedure for Atrial Fib;<br /><b>Performed</b>:2020-07-16T02:55:00-04:00;<br /><b>Focal Device</b>: <br /><b>Action</b>: <b>system</b>:http://snomed.info/sct <b>code</b>:386746003 <b>display</b>:Maintenance - action;<br /><b>Manipulated</b>:Device/7DD361F7EB43448690AABCCC45E62285.3007;<br /></div>'
+        # rubocop:enable Layout/LineLength
+      }
+    }.freeze
+
+    SOARIAN_R4_PROCEDURE_STOPPED ||= {
```
Comment (ShrutiNHegde) [inline lib/resources/example_json/soarian_r4_examples_procedure.rb:294]: I do not see any request URL is using this response in procedure.md file. Can we add it?

File: content/soarian/r4/clinical/summary/procedure.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+# Procedure
+
+* TOC
+{:toc}
+
+## Overview
+
+The Procedure resource implementation is limited to a patient’s known procedure history that occured at this site and is recorded in Soarian Clinicals<sup>®</sup>. As such, this resource may not be as complete as a procedure list provided as part of the patient’s billing record.
+
+Soarian Clinicals<sup>®</sup> supports a read-only Application Programming Interface (API). This API accepts `GET` and `POST` based [search] and `GET` based [read] interactions. The response represents the most current information about the patient that is charted in Soarian Clinicals<sup>®</sup> at the time of the query.
```
[the remark this answers is not in the document]
Comment (ShrutiNHegde) [reply content/soarian/r4/clinical/summary/procedure.md:14]: Can we add one example for POST?

**Review by HRNWriter — APPROVED** [review]
+1

**Review by ShrutiNHegde — APPROVED** [review]

**Review by slvsridhar — APPROVED** [review]

**Pull request merged**